### PR TITLE
Fix: Privileges from `adapta-gtk-theme` -> `micro`

### PR DESCRIPTION
**Pull request by wizard-28**
This fixes privilege for pacscripts from `adapta-gtk-theme` to `micro`, according to v1.4's new privilege policy

### CODE REVIEW

**Comment by wizard-28** [comment]
@pacstall/alacritty 
@pacstall/bat 
@pacstall/bpytop-git 
@pacstall/discord-canary 
@pacstall/discord 
@pacstall/exa 
@pacstall/gtkhash 
@pacstall/htop 
@pacstall/kvantum 
@pacstall/lagrange 

Review these changes

**Review by Henryws — APPROVED** [review]
looks good

File: packages/alacritty/alacritty.pacscript [hunk 1]
```diff
@@ -24,7 +24,7 @@ install() {
         sudo install -D extra/completions/_alacritty -t /usr/src/pacstall/${name}/usr/share/zsh/vendor-completions
 
         sudo mkdir -p /usr/src/pacstall/${name}/usr/local/share/man/man1
-        gzip -c extra/alacritty.man | sudo tee /usr/src/pacstall/${name}/usr/local/share/man/man1/alacritty.1.gz >/dev/null
+        sudo gzip -c extra/alacritty.man | sudo tee /usr/src/pacstall/${name}/usr/local/share/man/man1/alacritty.1.gz >/dev/null
```
Comment (kaistian) [inline packages/alacritty/alacritty.pacscript:27]: Why do you need sudo privileges to read the file `extra/alacritty.man`?
It's good security practice to limit the use of sudo to a absolute minimum.

Comment (wizard-28) [reply packages/alacritty/alacritty.pacscript:27]: Yeah, I just tested it, the `sudo` is not needed, thanks for informing!

Comment (wizard-28) [reply packages/alacritty/alacritty.pacscript]: Fixed it for both `alacritty` and `bat`

**Comment by wizard-28** [comment]
Merging this as 1.4 is already released

**Pull request merged**